src/components/rating/Rating.jsx

```diff
@@ -21,25 +25,51 @@ class Rating extends Component {
     if (ratedStarIndex !== rate) {
       onChange(ratedStarIndex);
     }
-  }
+  };
+
+  onMouseEnter = () => {
+    const {active} = this.props;
+
+    if (!active) {
+      return;
+    }
+
+    this.setState({showActiveText: true});
+  };
+
+  onMouseLeave = () => {
+    this.setState({showActiveText: false});
+  };
 
   render() {
-    const {metricSize = 5, rate = 0, small, active, counter, className} = this.props;
+    const {metricSize = 5, rate = 0, small, active, className, counterText, activeText} = this.props;
+    const {showActiveText} = this.state;
     const ratingClass = classnames('sg-rate-box', {
       'sg-rate-box--small': small,
       'sg-rate-box--active': active
     }, className);
+
     const starsProps = new Array(metricSize).fill(true).map((dump, index) => ({
```

Comment:
TIL about fill! Are you sure it's supported / polyfilled?

Reply:
it's default polyfill from polyfill.io